.gitignore

```diff
@@ -38,3 +38,4 @@ media/js/libs/ckeditor/source/ckbuilder
 .tox
 .env
 *_devmo.sql*
+.idea/*
```

Comment:
Please don't add such IDE specific items to the repo's gitignore. Instead I would recommend using a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).